content/en/observability_pipelines/_index.md

```diff
@@ -44,7 +44,7 @@ Datadog recommends you update Observability Pipelines Worker (OPW) with every mi
 
 ## Overview
 
-{{< img src="observability_pipelines/op_marketecture_08232024.png" alt="A graphic showing different data sources on the left that flows into three hexagons named transform, reduce, and route, with arrows pointing to different destinations for the modified data" style="width:100%;" >}}
+{{< img src="observability_pipelines/op_marketecture_11042024.png" alt="A graphic showing different data sources on the left that flows into three hexagons named transform, reduce, and route, with arrows pointing to different destinations for the modified data" style="width:100%;" >}}
```

Comment:
```suggestion
{{< img src="observability_pipelines/op_marketecture_11042024.png" alt="A graphic showing data being aggregated from a variety of sources, processed and enriched by the observability pipelines worker in your own environment, and then being routed to the security, analytics, and storage destinations of your choice" style="width:100%;" >}}
```

Reply:
Maybe I'm missing or misunderstanding something, but I couldn't find `three hexagons named transform, reduce, and route` in the image. This suggestion borrows from the language used in the arrows on the bottom of the diagram.

Reply:
Thanks for the suggestion! They had changed the diagram so it no longer has the three hexagons, but I forgot to update the alt text.